modules/node-certificates.adoc

```diff
@@ -0,0 +1,29 @@
+
+// Module included in the following assemblies:
+//
+// * authentication/understanding-certificates.adoc
+
+[id="node-certificates_{context}"]
+= Node certificates
```

Comment:
@rphillips @sjenning Can you please review what is here for node certs so far? Thanks!

Reply:
@rphillips @sjenning Can you please provide feedback by end of week?